doc/wui/ReleaseNotes.md

```diff
@@ -0,0 +1,7 @@
+Web User Interface V1
```

Comment:
wasn't there a request for change of the file name ReleaseNotes.md -> release_notes.md already?

Reply:
somehow renamed in some merges, it will be renamed